apps/retrospective/services/sprint_task.go

```diff
@@ -257,6 +258,7 @@ func (service SprintTaskService) MarkUndone(
 		Where("tasks.deleted_at IS NULL").
 		Where("id = (?)", query).
 		Update("done_at", nil).
+		Updates(map[string]interface{}{"done_at": nil, "resolution": nil}).
```

Comment:
It should be either this,
```suggestion
		Updates("resolution": nil).
```
...or remove the line above.

Reply:
done